platform/bukkit/src/main/java/me/lokka30/treasury/plugin/bukkit/hooks/miniplaceholders/MiniPlaceholdersHook.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * This file is/was part of Treasury. To read more information about Treasury such as its licensing, see <https://github.com/ArcanePlugins/Treasury>.
+ */
+
+package me.lokka30.treasury.plugin.bukkit.hooks.miniplaceholders;
+
+import io.github.miniplaceholders.api.Expansion;
+import java.io.File;
+import java.util.Locale;
+import java.util.UUID;
+import me.lokka30.treasury.plugin.bukkit.BukkitTreasuryPlugin;
+import me.lokka30.treasury.plugin.bukkit.vendor.BukkitVendor;
+import me.lokka30.treasury.plugin.core.hooks.Hook;
+import me.lokka30.treasury.plugin.core.hooks.PlayerData;
+import me.lokka30.treasury.plugin.core.hooks.placeholder.BasicPlaceholderExpansion;
+import me.lokka30.treasury.plugin.core.hooks.placeholder.PlaceholdersConfig;
+import me.lokka30.treasury.plugin.core.hooks.placeholder.minipspecific.MiniPlaceholdersConfig;
+import net.kyori.adventure.identity.Identity;
+import net.kyori.adventure.text.minimessage.tag.Tag;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class MiniPlaceholdersHook implements Hook {
+
+    private BasicPlaceholderExpansion basicExpansion;
+
+    @Override
+    public @NotNull String getPlugin() {
+        return "MiniPlaceholders";
+    }
+
+    @Override
+    public boolean register() {
+        if (!BukkitVendor.isPaper() && !BukkitVendor.isFolia()) {
+            return false;
+        }
```

Comment:
I don't think there is a need for this check. It is impossible for MiniPlaceholders to be initialized in a bukkit-based environment other than Paper, Folia or a fork of Paper